docs/reference/extensions/flutter-extension.rst

```diff
@@ -0,0 +1,148 @@
+.. _flutter-extension:
+
+Flutter extension
+=================
+
+The Flutter family of extensions help fill in common settings for software built with
+the `Flutter <https://flutter.dev>`__ framework.
+
+The Flutter extensions are currently only supported with the core18 base. Snaps using
```

Comment:
This means the flutter extensions only work with snapcraft 7 since [snapcraft 8 removes core18](https://github.com/canonical/snapcraft/releases/tag/8.0.0).

Reply:
Thanks @mr-cal for saving me an issue report! https://github.com/canonical/snapcraft/issues/5198

Reply:
l forgot the extension was superseded by the plugin in core20.  Should we even be adding this extension to the documentation?

Reply:
I'd say for now we put this in and say it only works with snapcraft 7 and we can remove this document with snapcraft 9. That way the documentation will still exist when the Discourse docs go away.

Reply:
Mentioned Snapcraft 7: [e37609e](https://github.com/canonical/snapcraft/pull/5226/commits/e37609e5b9cacc79c500f7e7c05c4e2d3ff2c981)

Reply:
Sounds good to me